core/Archiver/Request.php

```diff
@@ -68,9 +68,34 @@ public function setUrl($url)
     }
 
     public function changeDate($newDate)
+    {
+        $this->changeParam('date', $newDate);
+    }
+
+    public function makeSureDateIsNotSingleDayRange()
```

Comment:
Not sure if this changed in recent commit... shouldn't `setUrl` basically call this method?

Reply:
I thought about that then thought it might be a bad idea to do it every time the request url is set. Seems like it could cause strange side effects, and if we did, we'd have to check for an already running job after every setUrl() call.

Reply:
I can move it back if needed, though.

Reply:
Shouldn't be needed I reckon 👍